packages/eslint-plugin/src/rules/ngmodule-array-sorted.ts

```diff
@@ -0,0 +1,80 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { createESLintRule } from '../utils/create-eslint-rule';
+import { MODULE_CLASS_DECORATOR } from '../utils/selectors';
+import {
+  getDecoratorPropertyValue,
+  isArrayExpression,
+  isIdentifier,
+} from '../utils/utils';
+
+import { Options } from '../utils/property-selector';
+import { NgModule } from '@angular/compiler/src/core';
+
+export const RULE_NAME = 'ngmodule-array-sorted';
+export type MessageIds = 'sortedFailure';
+const validProperties: (keyof NgModule)[] = [
+  'bootstrap',
+  'declarations',
+  'entryComponents',
+  'exports',
+  'imports',
+  'providers',
+];
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'NgModule property arrays should be alphabetized for easy visual scanning.',
```

Comment:
@rafaelss95 

*NgModule **metadata** arrays

I think calling them "property" arrays is too ambiguous because of the NgModule class possible having properties

Reply:
@blingerson I guess you missed this suggestion :)